integration_tests/samples/socket_mode/bolt_oauth_example.py

```diff
@@ -0,0 +1,57 @@
+# ------------------
+# Only for running this script here
+import sys
+from os.path import dirname
+
+sys.path.insert(1, f"{dirname(__file__)}/../../..")
+# ------------------
+
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+
+import os
+
+from slack_bolt.app import App
+from slack_bolt.context import BoltContext
+from slack_bolt.oauth.oauth_settings import OAuthSettings
+
+app = App(
+    signing_secret=os.environ["SLACK_SIGNING_SECRET"],
+    oauth_settings=OAuthSettings(
+        client_id=os.environ["SLACK_CLIENT_ID"],
+        client_secret=os.environ["SLACK_CLIENT_SECRET"],
+        scopes=os.environ["SLACK_SCOPES"].split(","),
+    )
+)
+
+
+@app.event("app_mention")
+def mention(context: BoltContext):
+    context.say(":wave: Hi there!")
+
+
+@app.event("message")
+def message(context: BoltContext, event: dict):
+    context.client.reactions_add(
+        channel=event["channel"], timestamp=event["ts"], name="eyes",
+    )
+
+
+if __name__ == "__main__":
+
+    from bolt_adapter.websocket_client import SocketModeApp
```

Comment:
As websocket-client does not rely on asyncio at all, you can simply have OAuth flow endpoints and Socket Mode connection at the same time.